src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/dcoff.cpp

```diff
@@ -329,6 +329,14 @@ bool DCOFFPassBase::matcher_callback(ov::pass::pattern::Matcher& m) {
     NPUW_ASSERT(ov::op::util::is_parameter(matched_nodeC));
 
     auto matched_paramA = std::static_pointer_cast<ov::op::v0::Parameter>(matched_nodeA);
+    // Transpose weights specifically for QWEN as of now.
+    if (getTransposeWeights()) {
```

Comment:
not sure if getter is required here, in other cases passes just refer to their member variables.

Reply:
Changed!